tests/DCPS/SkipSerialize/run_test.pl

```diff
@@ -0,0 +1,142 @@
+eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
```

Comment:
You have to add this file to `bin/dcps_tests.lst` if you want it to be ran. Also please get rid of the configuration and security files that you won't need.

Reply:
I'll look into the `bin/dcps_tests.lst`. 

My plan was to only use the `rtps_disc` option from `run_test.pl` and get rid of everything else related to the other test scenarios. I hesitated thinking the other transports might be nice to keep but I think they are well covered in other tests. I'll get rid of the other test scenarios.

Reply:
Yea, unless there's something specific to the other transports or InfoRepo, RTPS should be fine.